Source/Blazorise.AntDesign/Styles/_switch.scss

```diff
@@ -0,0 +1,37 @@
+﻿@import "variables";
+
+.ant-switch-primary.ant-switch-checked {
+	background-color: $primary !important;
+}
+
+.ant-switch-secondary.ant-switch-checked {
+	background-color: $secondary !important;
```

Comment:
Try not to use `!important` whenever you can. It can be problematic to override color in user apps.

Reply:
Problem is, that color is elsewhere defined and i do not know where and whether i must delete it.
Like here:
![grafik](https://user-images.githubusercontent.com/56487025/86647840-aaf53e80-bfe0-11ea-9cc9-11156a6c4f20.png)
Last color is my color.

Reply:
It's probably set inside of theme generator.